src/game.cpp

```diff
@@ -201,13 +219,12 @@ bool Game::blockFits()
 void Game::reset()
 {
     grid.initialize();
-    blocks = getAllBlocks();
```

Comment:
We do not need this here any longer. getAllBlocks() now _internally_ holds **static storage**  to all blocks and returns a **const reference** to this permanent storage!